firmware/fome_generated.mk

```diff
@@ -5,9 +5,14 @@ GENERATED := \
 # $(GENERATED_DIR)/log_fields_generated.h \
 # $(GENERATED_DIR)/output_lookup_generated.cpp \
 # $(GENERATED_DIR)/rusefi_generated.h \
-# $(GENERATED_DIR)/value_lookup_generated.cpp
+# $(GENERATED_DIR)/value_lookup_generated.cpp \
+# $(PIN_NAMES_FILE)
 # TODO: how do we list multiple dependencies without the build happening multiple times?
```

Comment:
I'm not sure I understand "build running multiple times" ... is this the "two runs to stabilize Make" concept I've described, or different?